docker/cluster/submit_job.sh

```diff
@@ -1,11 +1,14 @@
 #!/usr/bin/env bash
```

Comment:
This script is becoming a bit messy if you want to handle both cases in one. Suggestion, make an individual submit_job_pbs.sh file and we only need a single if else statement in the container.sh file about which file to call. This should make is overall much simpler.